src/algorithms.rs

```diff
@@ -1,4 +1,5 @@
 use std::str::FromStr;
+use self::Algorithm::*;
```

Comment:
Yeah hyperglobs are nice for reexporting but this makes me uneasily flashback to my `using namespace` days :P